modules/account-lib/src/coin/algo/transactionBuilder.ts

```diff
@@ -258,6 +261,53 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     }
   }
 
+  /** @inheritdoc */
+  protected async buildImplementation(): Promise<Transaction> {
+    if (this._keyPairs.length === 0) {
+      throw new SigningError('No keypairs assigned for signing the transaction');
```

Comment:
Keep in mind that we may want to build an unsigned transaction at some point, for instance, to send to an offline signing tool.